### DISCUSSION TITLE
Channel Guide for IPTV

### DISCUSSION THREAD

**Discussion by JhonyBravoX** [iptv-org/community#1099] [Q&A]
Kindly can anybody tell me full guide. How to use this. Like I'm using Xstream Ui Iptv panel, I want to add but don't know where to add these. I have admin panel. Kindly guide me fully.

**Comment by freearhey** [comment]
We don't currently provide a guide, but you can create your own by following this instructions: https://github.com/iptv-org/epg?tab=readme-ov-file#installation
